src/main/java/org/railwaystations/api/resources/StationsResource.java

```diff
@@ -71,4 +104,21 @@ public Station getById(@PathParam(StationsResource.COUNTRY) final String country
         return stationMap;
     }
 
+    private String getClientLanguage(final HttpHeaders httpHeaders) {
+        if (httpHeaders.getAcceptableLanguages().isEmpty()) {
+            return DEFAULT_LICENSE_LANGUAGE;
+        } 
+        final Locale locale = httpHeaders.getAcceptableLanguages().get(0);
```

Comment:
According to the documentation the `getAcceptableLanguages()` can return a single Entry with Language set to `*`. How is this handled?